Thanks, the test failure (in circleCI) is real:
```
/home/circleci/repo/doc/source/reference/random/multithreading.rst:110: WARNING: Error in "code-block" directive:
maximum 1 argument(s) allowed, 26 supplied.

.. code-block:: ipython
    In [7]: # simulate the behavior for `threads=None`, if the machine had only one thread
       ...: mrng = MultithreadedRNG(10000000, seed=12345, threads=1)
       ...: print(mrng.values[-1])
    Out[7]: 1.1800150052158556
Makefile:178: recipe for target 'html-build' failed
```
My best (quick guess) is that it has incorrect indentation?

--- a/doc/source/reference/random/multithreading.rst
+++ b/doc/source/reference/random/multithreading.rst
@@ -19,7 +19,7 @@ change.
 
 .. code-block:: ipython
 
-    from numpy.random import default_rng, PCG64, SeedSequence
+    from numpy.random import default_rng, SeedSequence
     import multiprocessing
     import concurrent.futures
     import numpy as np

--- a/doc/source/reference/random/multithreading.rst
+++ b/doc/source/reference/random/multithreading.rst
@@ -108,6 +108,7 @@ Note that if `threads` is not set by the user, it will be determined by
 `multiprocessing.cpu_count()`.
 
 .. code-block:: ipython
+
     In [7]: # simulate the behavior for `threads=None`, if the machine had only one thread
        ...: mrng = MultithreadedRNG(10000000, seed=12345, threads=1)
        ...: print(mrng.values[-1])

